### DISCUSSION TITLE
GPU device is not detected inside openvino environment

### DISCUSSION THREAD

**Discussion by Nouran-Muhammad** [openvinotoolkit/openvino#16550] [Google Summer of Code]
Hello,
I have been working on the prerequisite task on ESPNet model. I created an openvino environment and installed all the requirements according to anaconda installation instructions [here](https://github.com/openvinotoolkit/openvino_notebooks/wiki/Conda). I cloned the model's repo and I have been trying to run inference using the pytorch version of the model or even load the model. It gives me an error saying that **torch.cuda.is_available() is False**.  CUDA cannot be detected although it can be detected in the base environment and any other environment I have created before. I have tried a lot with different versions but that would result in dependencies conflict.
I have uninstalled and reinstalled openvino in case any packages didn't install correctly but it didn't work.
Can someone tell me what to do?
Thanks

**Comment by adrianboguszewski** [comment]
Already answered here: https://github.com/openvinotoolkit/openvino_notebooks/issues/949
